spaceship/lib/spaceship/client.rb

```diff
@@ -603,6 +604,7 @@ def with_retry(tries = 5, &_block)
     rescue \
         Faraday::Error::ConnectionFailed,
         Faraday::Error::TimeoutError, # New Faraday version: Faraday::TimeoutError => ex
+        BadGatewayError,
```

Comment:
Is BadGatewayError really a "timeout" error?

Reply:
Maybe no. But it seems to occur temporarily.
It should be ignored or retried.